src/components/input/checkValidation.js

```diff
@@ -0,0 +1,35 @@
+import { GAME_MESSAGE } from '../../constants';
+
+const checkRange = (userNumber) => {
+  let flag = true;
+  for (let i = 0; i < userNumber.length; i++) {
+    if (userNumber[i] <= 0 && userNumber[i] > 9) flag = !flag;
+  }
+  return flag;
+};
+
+const checkLength = (userNumber) => {
+  if (userNumber.length === 3) return true;
+  else return false;
+};
+
+const checkUniqueness = (userNumber) => {
+  if (userNumber.length === [...new Set(userNumber)].length) return true;
```

Comment:
```suggestion
  if (userNumber.length === new Set(uerNumber).size) return true;
```

위에 처럼 Set을 배열로 바꾸지 않고 size 프로퍼티로 Set 내의 요소 개수를 가져올 수 있겠네요